### PR TITLE
Sphinx update

### PR DESCRIPTION
**Pull request by clairekope**
Should fix CircleCI "warning as error" failure for documentation build. Addresses a deprecation warning for Sphinx 7.

### CODE REVIEW

**Review by bwoshea — APPROVED** [review]

**Comment by bwoshea** [comment]
Looks good to me!

File: .gitignore [hunk 1]
```diff
@@ -56,3 +56,7 @@ doc/manual/build
 doc/manual/source/_build
 
 run/results.js
+
+Make.mach.claire-linux
```
Comment (gregbryan) [inline .gitignore:60]: Are these .gitignore changes needed?

**Review by gregbryan — APPROVED** [review]
One minor query, but otherwise looks good. Thanks!

File: .gitignore (hunk 1 above)
Comment (clairekope) [reply .gitignore:60]: Whoops, that snuck in there. I had it on the main of my fork and I'll pull it out in a later PR.

**Pull request merged**